app/views/entities/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<div>
+  <%= form_with(model: @entity, url: user_group_entities_path(current_user)) do |form| %>
+    <% if entity.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(entity.errors.count, "error") %> prohibited this entity from being saved:</h2>
+
+        <ul>
+          <% entity.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div>
+      <% form.label :name, style: "display: block" %>
+      <%= form.text_field :name, class:"mx-auto w-100 p-2", placeholder: "Name" %>
+    </div>
```

Comment:
### Add a new transaction page

- [ ]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻 
- [ ] It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one).  kindly prevent the user to create a new transaction without selecting at least one  categories 

![image](https://user-images.githubusercontent.com/79058364/234995505-67351f36-166e-4d86-9bad-a39fc7d93050.png)